### ISSUE TITLE
terraform error: Error: Failed creating the public key: Invalid PEM on the SSH private key

### ISSUE THREAD

**Issue by merryHunter** [iterative/cml#1116]
Hi, after yesterday release our gitlab pipeline failed with the following command:
```
{"level":"info","message":"Preparing workdir /home/runner..."}
{"level":"info","message":"Deploying cloud runner plan..."}
{"level":"info","message":"Terraform apply..."}
{"level":"info","message":"Terraform 1.2.6"}
{"level":"info","message":"iterative_cml_runner.runner: Plan to create"}
{"level":"info","message":"Plan: 1 to add, 0 to change, 0 to destroy."}
{"level":"info","message":"iterative_cml_runner.runner: Creating..."}
{"level":"info","message":"iterative_cml_runner.runner: Creation errored after 1s"}
{"level":"error","message":"terraform error: Error: Failed creating the public key: Invalid PEM on the SSH private key: []byte{0x2d, 0x2d, 0x2d, 0x2d, 0
....

2022-08-03T07:16:21.657Z [DEBUG] provider: plugin exited
2022-08-03T07:16:21.663Z [INFO]  provider: configuring client automatic mTLS
2022-08-03T07:16:21.697Z [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/iterative/iterative/0.11.4/linux_amd64/terraform-provider-iterative_v0.11.4 args=[.terraform/providers/registry.terraform.io/iterative/iterative/0.11.4/linux_amd64/terraform-provider-iterative_v0.11.4]
2022-08-03T07:16:21.698Z [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/iterative/iterative/0.11.4/linux_amd64/terraform-provider-iterative_v0.11.4 pid=141
2022-08-03T07:16:21.698Z [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/iterative/iterative/0.11.4/linux_amd64/terraform-provider-iterative_v0.11.4
2022-08-03T07:16:21.753Z [INFO]  provider.terraform-provider-iterative_v0.11.4: configuring server automatic mTLS: timestamp=2022-08-03T07:16:21.752Z
2022-08-03T07:16:21.771Z [DEBUG] provider: using plugin: version=5
2022-08-03T07:16:23.211Z [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = transport is closing"
2022-08-03T07:16:23.216Z [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/iterative/iterative/0.11.4/linux_amd[64](https://gitlab.com/<XXX>/jobs/11387#L64)/terraform-provider-iterative_v0.11.4 pid=141
2022-08-03T07:16:23.216Z [DEBUG] provider: plugin exited
{"level":"error","message":"terraform apply error","stack":"Error: terraform apply error\n    at Object.apply (/usr/lib/node_modules/@dvcorg/cml/src/terraform.js:54:11)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async runTerraform (/usr/lib/node_modules/@dvcorg/cml/bin/cml/runner.js:1[66](https://gitlab.com/<XXX>/jobs/11387#L66):5)\n    at async runCloud (/usr/lib/node_modules/@dvcorg/cml/bin/cml/runner.js:175:19)\n    at async run (/usr/lib/node_modules/@dvcorg/cml/bin/cml/runner.js:414:14)\n    at async Object.exports.handler (/usr/lib/node_modules/@dvcorg/cml/bin/cml/runner.js:428:5)","status":"terminated"}
```
The cml runner command we use: `cml-runner --cloud aws --cloud-region us-east-1 --cloud-type g4dn.2xlarge --labels=cml-runner-gpu --cloud-ssh-private="$(cat key.pem) "`

**Comment by 0x2b3bfa0** [comment]
Regression introduced by #1100, fixed with #1117 and ready for release.

**Comment by dacbd** [comment]
Released https://github.com/iterative/cml/releases/tag/v0.17.2
